include/ogdf/internal/decomposition/TricComp.h

```diff
@@ -85,7 +101,10 @@ class TricComp
 	//! number of components
 	int m_numComp;
 
-	//! check if computed triconnected componets are correct
+	/**
+	 * Checks if computed triconnected componets are correct.
+	 * \warning checkComp() assumes that the graph is simple!
```

Comment:
Use `\pre` (Precondition) instead. See [here](https://github.com/ogdf/ogdf/blob/2783d96540afc56b77308e5a7958078dfd354b00/include/ogdf/graphalg/MinCostFlowReinelt.h#L63-L64) for an example.